packages/client/src/links/internals/contentTypes.ts

```diff
@@ -0,0 +1,15 @@
+export function isOctetType(input: unknown) {
+  return (
+    input instanceof Uint8Array ||
+    input instanceof Blob ||
+    input instanceof File
```

Comment:
File is actually a Node 20 thing I think? Should be fine removing since it will be caught in `instanceof Blob` anyways so it's unreachable code

Reply:
Good catch, you're completely right!

<img width="515" alt="image" src="https://github.com/trpc/trpc/assets/8896153/d47a5af9-a4f6-429d-9138-b9cfde38ba15">

Reply:
and in browser Blob will be fine too:

<img width="285" alt="image" src="https://github.com/trpc/trpc/assets/8896153/7c5c952b-beeb-4f13-bf17-5e90053a2eaa">

Reply:
Pushed